### DISCUSSION TITLE
Defining https://lmfit.github.io as default

### DISCUSSION THREAD

**Discussion by Anselmoo** [lmfit/lmfit-py#846] [Ideas]
Currently, google _lmfit_ + _GitHub_ ends up to https://lmfit.github.io with **404** error.

So maybe change from  https://lmfit.github.io/lmfit-py/ to https://lmfit.github.io ? Would be a good idea?

<img width="833" alt="image" src="https://user-images.githubusercontent.com/13209783/222924181-f87bd812-d02e-49aa-9aad-271fb49d6354.png">

**Comment by newville** [comment]
@Anselmoo For me,  https://www.google.com/search?q=lmfit+github has the top 5 results of:
https://github.com/lmfit/lmfit-py
https://github.com/lmfit
https://github.com/lmfit/lmfit-py/blob/master/lmfit/models.py
https://lmfit.github.io/lmfit-py/
https://lmfit.github.io/lmfit-py/installation.html

That seems acceptable to me...

Typically, https://lmfit.github.io/ would be for the organization https://github.com/lmfit.  
We could add a simple doc for the organization to point to docs for specific projects within that organization.

**Reply by Anselmoo** [reply]
> Typically, https://lmfit.github.io/ would be for the organization https://github.com/lmfit.
> We could add a simple doc for the organization to point to docs for specific projects within that organization.

I like that idea a lot!
